benchmarking/frameworks/framework_base.py

```diff
@@ -54,6 +54,11 @@ def runBenchmark(self, info, benchmark, platform):
             for of in test["output_files"]:
                 result_files[of] = platform.getOutputDir() + "/" + of + ".txt"
 
+        if "preprocess" in test:
+            # simple thing first, let's assume preprocess is self contained
+            preprocess_cmd = self.composePreprocessCommand(test["preprocess"])
+            self.runPreprocessOnPlatform(preprocess_cmd, platform)
```

Comment:
maybe we should refactor it to be cleaner sometime, but we also need to copy the files to the platform.

Reply:
Yes, forgot to do it for mobile.